2022.md

```diff
@@ -1,6 +1,8 @@
 # 2022 Blogs
 
-## October (07)
+## October (09)
+* [Running WSO2 Products on Azure with Azure Membership Scheme](https://towardsdev.com/running-wso2-products-on-azure-with-azure-membership-scheme-3175113d8e10) by [Pabasara Mahindapala](https://medium.com/@pabasaramahindapala/)
+* [Monitoring Java Applications on Azure with Application Insights](https://medium.com/@pabasaramahindapala/monitoring-java-applications-on-azure-with-application-insights-246d6758337a) by [Pabasara Mahindapala](https://medium.com/@pabasaramahindapala/)
```

Comment:
Shall we add the two blogs at the end without adding it to the beginning?

Reply:
Sure. I will change it